integration/groupby_query_test.go

```diff
@@ -0,0 +1,85 @@
+// +build integration
```

Comment:
Rename this file to `groupby_query_no_filter_with_calculations_no_order.go`?

Reply:
I wanted to put many different types of group by queries in the same test file so I made the test file name generic.

Reply:
Spoke w/ xi offline, this is the standard pattern for writing integration tests.

Reply:
Changing the name to `groupby_query_no_filter_no_order.go` since calculations are required when you specify a group by condition so adding that to the name does not add any more information.